File: Easy-exercises/Paint Costs/Pain_Costs.cpp
#include<iostream>
#include<cmath>
#define canvas_brushes 40.00
using namespace std;
int main(void){
    float paint, res, tax, total;
    cout <<"Enter number of colors: " << endl;
    cin >>paint;
    res = ((paint * 5.00) + canvas_brushes);
    tax = ceil((res * 10) / 100);
    total = ceil(tax + res);
    cout << fixed;
    cout.precision(1);
    cout <<"Total cost for project is : " << total;
}

File: Easy-exercises/Paint Costs/Pain_Costs.php
<?php
define('CANVAS_BRUSHES', 40.00);
    echo ("Enter number of colors: \n");
    $paint = (float)readline("");
    $res = (($paint * 5.00) + CANVAS_BRUSHES);
    $tax = ceil(($res * 10) / 100);
    $total = ceil($tax + $res);
    echo ("Total cost for project is : ${total}");
?>